utils/load_path.mli

```diff
@@ -49,6 +52,9 @@ module Dir : sig
   val find_normalized : t -> string -> string option
   (** As {!find}, but search also for uncapitalized name, i.e. if name is
       Foo.ml, either /path/Foo.ml or /path/foo.ml may be returned. *)
+
+  val error_occurred : t -> string option
```

Comment:
This API is error-prone: it is easy to forget to call this function when needed. Instead, the function `create` should return a pair `t * string option` with the error.

Reply:
My concern was that current consumers of the `create` function simply ignore the error. Is it fine if I use `create path |> fst` in all current call sites?

Reply:
I think so. The difference is that the consumers now are _forced_ to ignore the error message (by ignoring the second component of the pair), as opposed to hiding the error in a side effect. Personally I would use pattern matching instead of composing with `fst`, but that's just my taste.